lib/CL/pocl_llvm_wg.cc

```diff
@@ -225,18 +226,9 @@ llvm::Error PoCLModulePassManager::build(std::string PoclPipeline,
   PassBuilder &PB = *PassB.get();
 
 #if 0
```

Comment:
Did you notice the block is still disabled?

Reply:
Yes, it's supposed to be. I've tried putting the createTLII(... vectorlibrary) here before i realized it only works in codegen.